WordPress/gutenberg
# Remove @storybook/addon-knobs from the repository

## What?


Remove the `@storybook/addon-knobs` dependency from the repository

## Why?


The dependency has been deprecated for a long time, and we recently removed all usages from all Storybook examples.

This dependency is currently blocking using npm 8 (see https://github.com/WordPress/gutenberg/issues/46443)

## How?


Removed from:
- npm dependency
- docs
- Storybook settings

## Testing Instructions





- Make sure that the project builds correctly
- Make sure that Storybook examples load and function as on `trunk`
- Do a smoke test on the editor, make sure that everything works as on `trunk`
